src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/Shower.java

```diff
@@ -0,0 +1,38 @@
+package eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities;
+
+/**
+ * Shower is part of the set of commodities
+ */
+public class Shower extends AbstractCommodity {
+
+    private boolean hasFloorHeating;
+
+    /**
+     * Parametrized constructor
+     *
+     * @param hasFloorHeating shower's extra
+     */
+    public Shower(boolean hasFloorHeating) {
+        super();
+        this.hasFloorHeating = hasFloorHeating;
+    }
+
+    public boolean HasFloorHeating() {
```

Comment:
method does not follow the spring bean convention. It should start with a lowercase letter